resources/js/Components/Form/EntryCreateForm.tsx

```diff
@@ -71,9 +74,13 @@ const EntryCreateForm = ({
     adventures,
     gameMasters,
     campaigns,
+    campaignEntry,
```

Comment:
actually here, add another string value in `type` instead, no need for an additional prop